docs/sql-migration-guide-upgrade.md

```diff
@@ -23,6 +23,9 @@ license: |
 {:toc}
 
 ## Upgrading From Spark SQL 2.4 to 3.0
+  - Since Spark 3.0, hive is upgraded to 2.3.x, so it is required to update the Hive
```

Comment:
Hive is only upgraded into 2.3.5 (and presumably it will be 2.3.6) only Hadoop 3 profile, which is not officially released with.